suse/rdma.sriov-service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Initialize SRIOV for RDMA devices
+RefuseManualStop=true
+DefaultDependencies=false
+Conflicts=emergency.target emergency.service
+Before=network.target remote-fs-pre.target
+
```

Comment:
The before's should be:

```
# Partially support distro network setup scripts that try to run after hardware is configured.
# Run them after any SRIOV devices have been created
Wants=network-pre.target
Before=network-pre.target
# RDMA is not ready until all SRIOV devices are created
Before=rdma-hw.target
```

Reply:
If this is triggered by udev, does it have an impact ?

But OK

Reply:
Yes, ordering is important! The sriov script may also need a 'udevadm settle' at the end to make the ordering predictable.